lib/app/backend/PipelineWatcher.js

```diff
@@ -11,11 +11,12 @@ class PipelineWatcher {
   constructor (appSettings) {
     this.appFolder = appSettings.getApplicationFolder()
     this.options = {ignoreInitial: true}
+    this.chokidar = chokidar
   }
 
   start () {
     return new Promise((resolve) => {
-      this.watcher = chokidar.watch(path.join(this.appFolder, EXTENSIONS_FOLDER, '**', 'pipelines', '*.json'), this.options)
+      this.watcher = this.chokidar.watch(path.join(this.appFolder, EXTENSIONS_FOLDER, '**', 'pipelines', '*.json'), this.options)
```

Comment:
Change '**' to '*' because pipelines have to be in $extensionFolder/$extension/pipelines